# awesome-advisors
Academic (Masters', PhD, Postdoc) advisors with explicitly stated expectations of themselves and their mentees in the form of written lab manuals. In general, this includes:
- Lab manual with expectations of Masters' and PhD students and Postdocs
- Work-life balance philosophies
- Code of Conduct with anti-harassment policies
- Culture of sharing, typically "open science"-focused, especially publicly sharing code and data

## Evolutionary Biology

* Carvunis, Anne at University of Pittsburgh School of Medicine, Pennsylvania, USA. [Lab Website](https://www.csb.pitt.edu/faculty/carvunis/). [Lab Manual](https://docs.google.com/document/d/1527btfgL-NcJiO3TZnjJ61VDmMiCJvuiunoA6XTi488/edit). [Twitter](https://twitter.com/carvunis).
  > What makes each species unique? In the Carvunis lab, we study the molecular mechanisms of change and innovation in evolution. This involves thinking about how genomes change over time, what cellular processes enable these changes, and how novel molecular networks emerge. We consider evolution in the light of systems biology and systems biology in the light of evolution, in order to gain a better understanding of how cells and organisms live and evolve. The research tools we rely on most are bioinformatics, yeast genetics and genomics. Generally, we strive to foster an interdisciplinary and collaborative research environment where researchers can develop creative approaches to describe, engineer and predict the genetic and network-level determinants of species-specificity.


* Cooper, Natalie at Natural History Museum, London, UK. [Lab Website](http://nhcooper123.github.io/). [Lab Manual](https://github.com/nhcooper123/macro-group-info). [Twitter](https://twitter.com/nhcooper123). [GitHub](https://github.com/nhcooper123).
  > Our research sits at the interface between macroevolution and macroecology, and aims to understand broad-scale patterns of biodiversity. We use cutting-edge phylogenetic comparative methods (usually in R), morphological data collected from museum specimens, phylogenies, ecological data, species geographic range maps, fossils and various larger databases, to investigate a variety of topics. Current research in the group includes analyses rates and modes of evolution, convergent evolution in various taxa (including tenrecs and river dolphins), macroevolutionary analyses with fossil and living species, responses of cetaceans to past and present climate change, and biogeography and conservation of amphibians. We are advocates for Open Science and diversity in STEM.


## Cellular Biology

* Avasthi, Prachee at University of Kansas Medical Center. [Lab Website](http://www.avasthilab.org/). [Lab Manual](http://www.avasthilab.org/lab-policies-and-tips/). [Twitter](https://twitter.com/PracheeAC)
  > Our lab studies mechanisms that regulate the normal and pathological functions of the nearly ubiquitous organelle, the cilium. This highly conserved cellular antenna, also known as a flagellum, requires coordination of the cell cycle, cytoskeletal dynamics, and intracellular trafficking for structural maintenance and signal transduction. Due to the role of the cilium in essential functions in nearly all human cells, abnormalities result in a wide range of diseases, termed ciliopathies. In fact, ciliary signaling is now understood to play a role in very common diseases such as diabetes and cancer and may provide new avenues for therapeutic intervention for these devastating disorders.
  
* Chan, C Savio at the Feinberg School of Medicine at Northwestern University. [Lab Website](https://www.saviochanlab.org/). [Lab Manual](https://www.saviochanlab.org/join). [Twitter](https://twitter.com/csaviochan). [GitHub](https://github.com/chanlab/)
  > Our research interests center on better understanding the cellular and molecular building blocks that make up the basal ganglia macrocircuit as well as their implications on neural activity level, pattern, and plasticity in the basal ganglia in health and disease. We hope to translate our insights into developing novel treatments for neurological disorders of the basal ganglia such as Parkinson's disease, dystonia, and Huntington's Disease. 

  
## Computational Biology

* Brown, C Titus at the Population Health and Reproduction department at University of California, Davis. [Lab Website](http://ivory.idyll.org/lab/index.html). [Lab Manual](http://ivory.idyll.org/lab/work-guidelines.html). [Twitter](https://twitter.com/ctitusbrown). [GitHub](https://github.com/dib-lab/)
  > The DIB Lab tackles questions surrounding biological data analysis, data integration, and data sharing. Our primary interest is in genomic, transcriptomic, and metagenomic sequence analysis.
  > In brief,
  > 
  > - the lab is the primary developer of the khmer software, for faster and more efficient sequence analysis of high-throughput sequencing data.
  > - we run quite a bit of training in data-intensive biology.
  > - we coordinate training and communication for the NIH-funded Data Commons Pilot Phase.
  > - the lab is located at the University of California, Davis in the School of Veterinary Medicine.
  
## Molecular Biology

* DePace, Angela at the Systems Biology Department at Harvard Medical School. [Lab Website](https://depace.med.harvard.edu/). [Lab Manual](https://depace.med.harvard.edu/?page_id=408). [Twitter](https://twitter.com/depacelab).
  > Our long-term goal is to understand how regulatory DNA dictates transcriptional network behavior and, ultimately, organismal phenotype and evolution. Our approach is mechanistically motivated: we believe that understanding the molecular mechanisms that drive transcription will lead to models of gene regulation that can predict the functional consequences of regulatory sequence changes and guide production of new types of regulatory circuits.




